### PR TITLE
Fix Ghidra serializing negative 64-bit addresses

### PR DESCRIPTION
**Pull request by rbs-afflitto**
- [x] I have reviewed the [OFRAK contributor guide](https://ofrak.com/docs/contributor-guide/getting-started.html) and attest that this pull request is in accordance with it.

**One sentence summary of this PR (This should go in the CHANGELOG!)**
* Fix Ghidra serializing negative 64-bit addresses

**Link to Related Issue(s)**
N/A

**Please describe the changes in your request.**
The OFRAK Ghidra scripts use `%d` string format of long integers when building JSON messages. In Java, this will be a signed decimal, but OFRAK expects an unsigned decimal. This causes problems in 64-bit Linux kernels, where commonly the address space is in the negative range. This change uses `Long.toUnsignedString` to format all long integers as unsigned before serializing to JSON. This PR also adds a test case which uses an elf with a negative address range.

**Anyone you think should look at this, specifically?**
@whyitfor @SamL98

### CODE REVIEW

**Review by whyitfor — CHANGES_REQUESTED** [review]
Please add a changelog to this MR

File: disassemblers/ofrak_ghidra/CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ All notable changes to `ofrak-ghidra` will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) and adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased](https://github.com/redballoonsecurity/ofrak/tree/master)
+### Fixed
+- Fixed handling negative 64-bit addresses. ([#474](https://github.com/redballoonsecurity/ofrak/pull/474))
```
Comment (whyitfor) [inline disassemblers/ofrak_ghidra/CHANGELOG.md:8]:
```suggestion
- Ensure large 64-bit addresses are interpreted as unsigned. ([#474](https://github.com/redballoonsecurity/ofrak/pull/474))
```

File: ofrak_core/pytest_ofrak/patterns/basic_block_unpacker.py [hunk 2]
```diff
@@ -1319,6 +1319,36 @@ class BasicBlockUnpackerTestCase(
             0x4005A0: Range(0x5A0, 0x5AF),
         },
     ),
+    # x64 Kernel address space
```
Comment (whyitfor) [inline ofrak_core/pytest_ofrak/patterns/basic_block_unpacker.py:1322]: Please update this comment to more explicitly state what bug/failure it is testing (see proposed changelog).

As a sanity check -- this test should **_FAIL_** without the change. Is this the case?

File: ofrak_core/pytest_ofrak/patterns/code_region_unpacker.py [hunk 3]
```diff
@@ -74,6 +74,16 @@ class CodeRegionUnpackerTestCase(UnpackAndVerifyTestCase[int, List[int]]):
         "hello_nosections.out",
         "56662f638390cae92c2cf5107bc3f1ef",
     ),
+    CodeRegionUnpackerTestCase(
```
Comment (whyitfor) [inline ofrak_core/pytest_ofrak/patterns/code_region_unpacker.py:77]: Similarly, let's make this more explicit about what this is testing.

**Review by whyitfor — CHANGES_REQUESTED** [review]
Looks good -- see a few minor comments/suggestions.

**Comment by SamL98** [comment]
LGTM. We should really use an actual JSON library in the ghidra scripts eventually.

File: ofrak_core/pytest_ofrak/patterns/basic_block_unpacker.py (hunk 2 above)
Comment (rbs-afflitto) [reply ofrak_core/pytest_ofrak/patterns/basic_block_unpacker.py:1322]: Yes, I've confirmed that these tests fail without the Ghidra script changes.

**Review by SamL98 — APPROVED** [review]

**Review by whyitfor — APPROVED** [review]
LGTM
